script.js

```diff
@@ -516,4 +516,24 @@ form.addEventListener('submit', (e) => {
     validation.innerHTML = messages.join(',');
   }
   e.preventDefault();
-});
\ No newline at end of file
+});
+
+let localInfo = {
+  fullName: '',
+  email: '',
+  comment: '',
+};
+
+if (localStorage.getItem('localInfo') != null) {
+  localInfo = JSON.parse(localStorage.getItem('localInfo'));
+  name.value = localInfo.fullName;
+  emailAddress.value = localInfo.email;
+  message.value = localInfo.comment;
+} else {
+  form.addEventListener('submit', () => {
+    localInfo.fullName = name.value;
+    localInfo.email = emailAddress.value;
+    localInfo.comment = message.value;
+    localStorage.setItem('localInfo', JSON.stringify(localInfo));
+  });
+}
```

Comment:
- [ ] The form is not being saved, after reload I am loosing my inputs https://www.awesomescreenshot.com/video/12603832

Reply:
I am not able to see the video you sent me, please can you screenshots??

Reply:
I cannot send you a screenshot because I wanted to show you that after re-loading the inputs are not being saved. You should be able to watch the video

Reply:
On my local environment, It is working very well.
can I record loom video??